pallets/claims/src/lib.rs

```diff
@@ -617,11 +650,20 @@ impl<T: Config> Pallet<T> {
 		Some(res)
 	}
 
-	fn process_claim(signer: EthereumAddress, dest: T::AccountId) -> sp_runtime::DispatchResult {
+	fn process_claim(
+		signer: EthereumAddress,
+		dest: Option<T::AccountId>,
+	) -> sp_runtime::DispatchResult {
+		println!("process_claim: signer: {:?}, dest: {:?}", signer, dest);
```

Comment:
```suggestion
```

Reply:
Thanks I'll remove these as soon as the logic is finished.